test/cli-validator/mockFiles/justWarnConfigOverride.json

```diff
@@ -0,0 +1,10 @@
+{
+  "shared": {
+    "operations": {
+      "no_operation_id": "off"
+    },
+    "schemas": {
+      "no_schema_description": "error"
+    }
+  }
+}
```

Comment:
Is this file going to be used? If not, will you remove it?

Reply:
Yes, I can remove it.  I wanted to provide only the options different than the defaults to eliminate having to maintain it if other options are added in the future.  I think this is the right behavior, but during test time it seems that this loading is not really working and that we have to provide all of the options.  In any case, I will remove this partial one for new and add it back if we fix the behavior of how the loading is done so it works during test.

Reply:
It turns out that this does in fact work, so I removed the justWarnConfigOverrideFull.json file instead and changed the test to use the partial one.

Reply:
Cool, I thought that should work. That looks good 👍